Dockerfile

```diff
@@ -1,12 +1,13 @@
-FROM docker.io/centos:centos7
+FROM docker.io/centos:centos8
 
-RUN yum install -y python-requests && \
-    curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/tripleo_repos/main.py | python - -b stein current-tripleo && \
+RUN yum install -y python3 python3-requests && \
```

Comment:
Do you really need to install python3 explicitly?

Reply:
it was necessary until last time :/